fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -3135,6 +3135,10 @@ public class Config extends ConfigBase {
             "occupying too much meta memory")
     public static int max_mv_task_run_meta_message_values_length = 16;
 
+    @ConfField(mutable = true, comment = "Whether enable to use list partition rather than range partition for " +
+            "all external table partition types")
+    public static boolean enable_mv_list_partition_for_external_table = false;
```

Comment:
why not enable it by default?

Reply:
This may change a lot tests and behaviors.

I want to enable it in the next pr to avoid this bugfix.